src/main/assembly/dist/cfg/config.yml

```diff
@@ -11,25 +11,22 @@ server:
       # Replace with port number unique for this service
       port: 20001
 
-
-
 easyDb:
-  # TODO: Modify for connecting to easy-db
-  driverClass: org.hsqldb.jdbcDriver
-    url: jdbc:hsqldb:hsql://localhost:9001/dd-manage-prestaging
-    logValidationErrors: true
-    # See: https://stackoverflow.com/questions/10684244/dbcp-validationquery-for-different-databases
-    validationQuery: SELECT * FROM INFORMATION_SCHEMA.SYSTEM_TABLES
-    properties:
-      hibernate.dialect: 'org.hibernate.dialect.HSQLDialect'
-      hibernate.hbm2ddl.auto: update
+  driverClass: org.postgresql.Driver
+  url: 'jdbc:postgresql://easy.dans.knaw.nl:5432/easy_db'
+  logValidationErrors: true
+  user: easy_webui
+  password: changeme
+  properties:
+    hibernate.dialect: 'org.hibernate.dialect.PostgreSQL92Dialect'
+    hibernate.hbm2ddl.auto: update
 
 verificationDatabase:
   driverClass: org.hsqldb.jdbcDriver
-    url: jdbc:hsqldb:hsql://localhost:9001/dd-verify-file-migration
-    logValidationErrors: true
-    # See: https://stackoverflow.com/questions/10684244/dbcp-validationquery-for-different-databases
-    validationQuery: SELECT * FROM INFORMATION_SCHEMA.SYSTEM_TABLES
-    properties:
-      hibernate.dialect: 'org.hibernate.dialect.HSQLDialect'
-      hibernate.hbm2ddl.auto: update
+  url: 'jdbc:postgresql://easy.dans.knaw.nl:5432/dd_verify_file_migration'
```

Comment:
Somewhere must be a mistake, after deploy and running on deasy (as specified in the cover message of this PR) I get a connection error, the stack trace does not give a clue on which DB.

     org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator: HHH000342: Could not obtain connection to query metadata : The connection attempt failed.

Reply:
localhost in the URL does improve things.